### PR TITLE
Add documentation for notification policies

### PR DESCRIPTION
**Pull request by ClearlyClaire**

### CODE REVIEW

File: content/en/methods/notifications.md [hunk 1]
```diff
@@ -391,6 +391,426 @@ Invalid or missing Authorization header.
...
+
+Authorization
+: {{<required>}} Provide this header with `Bearer <user token>` to gain authorized access to this API method.
+
+#### Response
+
+```http
+GET https://mastodon.social/api/v1/notifications/requests/112456967201894256 HTTP/1.1
+Authorization: Bearer xxx
+```
```
Comment (trwnh) [inline content/en/methods/notifications.md:652]: same

**Comment by trwnh** [comment]
Looks good overall but IDK if there should be a sample request under every `#### Response` header.

**Comment by ClearlyClaire** [comment]
Updated to remove the redundant HTTP requests examples and add missing documentation for the new entities.

**Comment by mcclure** [comment]
Can we use the presence/absence of the `/api/v1/notifications/policy` endpoint to detect the presence of this feature?

**Comment by ClearlyClaire** [comment]
> Can we use the presence/absence of the `/api/v1/notifications/policy` endpoint to detect the presence of this feature?

I think so, yes.

**Review by renchap — APPROVED** [review]

**Pull request merged**